section2/reflection.md

```diff
@@ -2,14 +2,35 @@
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+* A conditional statement runs through a set of `if` and `else if` conditions until it finds one that is true. JavaScript stops when it finds the true statement or reaches a `else` statement then used the code inside the statements code block for the the result.
+
+* I think that the most "Popular this Month" blog posts on CSS-Tricks may be filtered by conditional statement that checks how many views or likes a blog post has and if it meets a certain condition then it's added to the "Popular this Month" highlight.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+A conditional statement runs through the conditions of `if`,  `else if`, and `else` until it evaluates a true condition or it ends up at the `else`. JavaScript will first run `if`. If it's true JavaScript will stop in it's code block. If it's false JavaScript will move onto the next conditional statement, `else if`. If that is true JavaScript will stop there. If its false JavaScript will  moving onto the next statement until it finds a true outcome. If JavaScript reaches the `else` without finding a true outcome, then the code in the `else` code block will be what JavaScript uses.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+If you want to check that two conditions are met you would use the AND `&&` operator. If you want to see if one of two conditions are met you would use the OR `||` operator.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+I felt confident through this whole section until I got to the pizza problem in the "if-statement" exercises. I didn't understand how the logical operators worked when used with a variable that wasn't explicitly true or false. I found information on [JavaScript.info](https://javascript.info/logical-operators) that explained logical operators with examples using variables that helped me understand. I believe I found a good solution for the pizza problem, my conditional statement gets all the outcomes expected. I would like to know if there is a better way to write the statement or even other ways to write it.
```

Comment:
Your solution looks good to me, that is something that you could refactor later if working with more complex logic like so: 
```js
var cupsOfFlour = 2;
var hasSauce = true;

function hasEnoughFlour(cupsOfFlour) {
  if (cupsOfFlour >= 2){
    return true
  } else {
    return false
  }
}

if (hasEnoughFlour(cupsOfFlour) && hasSauce) {
  console.log("I can make pizza");
} else {
  console.log("I cannot make pizza");
}
```
I think for this problem your solution makes more sense, but as you get more conditions, a refactor like this might make more sense to avoid really long and complex conditional statements and to make it more readable.